tests/sources/test_tox.py

```diff
@@ -15,8 +15,11 @@
 from check_python_versions.versions import Version
 
 
-def v(versions: List[str]) -> List[Version]:
-    return [Version.from_string(v) for v in versions]
+def v(versions: List[str], has_dot: List[bool] = []) -> List[Version]:
+    if not has_dot:
+        has_dot = [False] * len(versions)
+    return [Version.from_string(v, has_dot=d)
+            for v, d in zip(versions, has_dot)]
```

Comment:
Why is this needed?  Is `Version(3, 11, has_dot=True) != Version(3, 11, has_dot=False)`?  No, that's going to be causing confusion and bugs.